### PR TITLE
HAWNG-646: Default connection modal to https scheme

### PR DESCRIPTION
**Pull request by phantomjinx**
* Makes the switch on the ConnectionModal default to https

### CODE REVIEW

**Comment by phantomjinx** [comment]
`https` selected by default:

Screenshot_20240508_133246

File: packages/hawtio/src/plugins/connect/remote/Remote.tsx [hunk 1]
```diff
@@ -61,6 +61,8 @@ const RemoteToolbar: React.FunctionComponent = () => {
   }
 
   const initialConnection = { ...INITIAL_CONNECTION }
+  // default connection modal to 'https'
+  initialConnection.scheme = 'https'
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/connect/remote/Remote.tsx:65]: why don't we change the INITIAL_CONNECTION directly instead?

Comment (phantomjinx) [reply packages/hawtio/src/plugins/connect/remote/Remote.tsx:65]: Did so initially but used elsewhere so was being cautious. Will change accordingly.

**Comment by phantomjinx** [comment]
@jsolovjo 
Looks like this broke the e2e tests?

**Comment by jsolovjo** [comment]
> @jsolovjo Looks like this broke the e2e tests?

seems yes
as the tests use simple http connection
I can fix it

**Review by tadayosi — APPROVED** [review]

**Pull request merged**